recipes/runjob/post-link.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python -m pip install batchcompute
```

Comment:
Please package this dependency and include it in the meta.yml

Reply:
Sorry, This package is a third-package from Pypi, but I do not find in Anaconda，So I include this in the post-link.sh， I don't know how to package this dependency. Could you give me some advice? thanks.

Reply:
As you did this package here. You need to create a recipe and probably need to submit it to conda-forge. Here are some instructions: https://conda-forge.org/#add_recipe

Thanks @yodeng

Reply:
OK，Thanks.  I'm not the author of that third-package, so I think it's not proper to submit that packages to conda-forge by me. I have update the recipe and include that dependency in meta.yml. The post-link.sh file has been also removed. Please check and review again.
Thank you so much @bgruening

Reply:
I don't understand. The requirement is not in the meta.yml and if there is no conda package it will also not work.
Its very normal that people maintain and create packages that they are not developing. This is the standard, so feel encouraged to create this missing package.

Reply:
I have add the requirement in build script section of meta.yml. The pip command will install my package and the url remote packages with ‘--no-deps’ option. Dependency will be install in build step as the same time. So Im sure it works, the reference example from [here](https://docs.conda.io/projects/conda-build/en/latest/user-guide/wheel-files.html?highlight=wheel).

Reply:
I'm not sure we can merge it like that. Please stick to the our usual recommendations and take inspiration from other python packages.

Reply:
Yes, I'm inspired by the [TensorFlow recipe](https://github.com/conda-archive/conda-recipes/blob/a796713805ac8eceed191c0cb475b51f4d00718c/python/tensorflow/build.sh#L7). It also installs the package from a remote url by pip command. It's the best way to include dependencies in a meta.yaml that are not yet available as conda packages.

Reply:
No TensorFlow does this because of different reasons. And TF is installing TF ... not additional packages.
Please do not mix pip and conda packages if you can avoid it.